meson.build

```diff
@@ -0,0 +1,61 @@
+project(
+  'screentest',
+  'c',
+  license: 'GPL-2.0-only',
+  license_files: ['COPYING'],
+  meson_version: '>= 1.1',
+  version: '2.2-dev',
+)
+
+i18n = import('i18n')
+
+gmodule_dep = dependency('gmodule-2.0', version: '>= 1.1.3')
+gtk_dep = dependency('gtk+-2.0', version: '>= 2.24')
```

Comment:
The `configure.ac` had _GTK+_ `2.16` as minimum requirement. But since _GTK+_ `2.24` is already more than ten years old and me intending to try a port to _GTK+_ `3` `2.24` seems like a sane new default.

Reply:
Sure, no problem. You might have noticed I didn't update this project for a long time, so those requirements list what was available ages ago.